backup_lab/lab07.md

```diff
@@ -16,31 +16,21 @@ The goal of this lab is get more practice with iterating through linked lists an
 
 ## Step 1: Getting Started
 
-1. Decide if you are working alone, or working in a pair. Pair programming is OPTIONAL for this lab.
 
-2. Choose who will be the first driver and who will start as navigator, and then remember to switch (at least once) during the lab.
+1. Go to github and create a git repo for this lab following the naming convention specified in previous labs (this step carries style points, see our feedback on previous labs to understand what we are looking for).l
 
-3. Go to github and create a git repo for this lab following the naming convention specified in previous labs (this step carries style points, see our feedback on previous labs to understand what we are looking for). If you are working with a partner only one of you needs to create the repo.
+2. Log on to your CSIL account and open a terminal window.
 
-4. If you are working with a partner and you are the one who created the github repo, add your partner as a collborator on the repo
-
-5. Decide on initial navigator and driver.
-
-6. Driver, log on to your CSIL account.
-
-7. Open a terminal window and log into the correct machine.
-
-8. Change into your CS 16 directory
-
-Note: Remember to push your work to github at the end of EVERY work session. That way, both partners always have access to the latest version of the code even if the code is being developed on one partner's CoE account.
+3. Change into your CS 16 directory.
 
+Note: Remember to push your work to github at the end of EVERY work session.
 
 
 ## Step 2: Obtaining the starter code
 
 * Navigate to your cs16 directory and clone the git repository you created
 ```
-git clone git@github.com:ucsb-cs16-s18-mirza/lab07_alily_jgaucho.git
+git clone git@github.com:ucsb-cs16-w19/lab07_alily_jgaucho.git
```

Comment:
Can these instructions be more synchronized with previous labs? The order seems to be a bit confusing. 

Perhaps, we can begin with the starter code, referring them to past labs for additional instructions.

Reply:
I think I have made the instructions more concise now.